packages/kit/src/vite/utils.js

```diff
@@ -235,7 +245,7 @@ const find_illegal_rollup_imports = (
 	illegal_imports,
 	seen = new Set()
 ) => {
-	const name = normalizePath(node.id);
+	const name = removeQueryFromPath(normalizePath(node.id));
```

Comment:
this suggestion got lost somehow

```suggestion
	const name = remove_query_from_path(normalizePath(node.id));
```